### PR TITLE
Split resources to named and groups

### PR DESCRIPTION
**Pull request by soltysh**
/assign @sferich888

### CODE REVIEW

**Comment by sferich888** [comment]
@mfojtik I think we still want to pus/fix this correct? 
/lgtm but we need it to have merge conflicts fixed.

**Comment by sferich888** [comment]
I think this PR is a way of addressing https://github.com/openshift/must-gather/issues/6  (however its taking an alternative approach, to what is discussed in that issue).

**Comment by sferich888** [comment]
This PR should obsolete https://github.com/openshift/must-gather/pull/88 as well. IE if this PR merges we need to close #88 as its not longer needed.

**Comment by soltysh** [comment]
@sferich888 rebased and ready for merge

**Comment by soltysh** [comment]
/hold
looks like the failure is real

**Comment by soltysh** [comment]
This PR actually found a bug in `oc adm inspect` when you specify multiple resources. Namely `oc adm inspect clusterversions clusteroperators` will fail and it should not. Working on a fix...

**Comment by soltysh** [comment]
Actually, it was my bad, when you're passing multiple group of resources you need to split them with a comma, (eg. `pods,services`), whereas named resources need to be split with a space (et. `pod/a pod/b`). I've updated help for `oc adm inspect` to include examples in https://github.com/openshift/oc/pull/621
This should be now green and ready to go.
/hold cancel

File: collection-scripts/gather [hunk 1]
```diff
@@ -5,37 +5,42 @@
...
-resources+=(networks.operator.openshift.io)
+group_resources+=(networks.operator.openshift.io)
 
 # Run the Collection of Resources using inspect
-for resource in "${resources[@]}"; do
-    oc adm inspect --dest-dir must-gather ${resource}
-done
+# running accross all-namespaces for the few "Autoscaler" resouces.
+oc adm inspect --dest-dir must-gather "${named_resources[@]}"
+group_resources_text=$(IFS=, ; echo "${group_resources[*]}")
```
Comment (sferich888) [inline collection-scripts/gather:42]: Why do we do this for grouped resources but not named resources? 
Having this here in the conversation would be good for posterity.

**Pull request merged**